mysql-test/suite/innodb/t/doublewrite_debug.test

```diff
@@ -57,8 +60,9 @@ set global innodb_saved_page_number_debug = 1;
 set global innodb_fil_make_page_dirty_debug = 0;
 
 set global innodb_buf_flush_list_now = 1;
-let $shutdown_timeout=0;
---source include/shutdown_mysqld.inc
+
+--let CLEANUP_IF_CHECKPOINT=drop table t1;
+--source ../include/no_checkpoint_end.inc
```

Comment:
I think that we want the test to fail rather than be skipped when a checkpoint occurs. Please define a statement that will cause an error, such as
```sql
--let CLEANUP_IF_CHECKPOINT=drop table t1,unexpected_checkpoint;
```